File: XPlaneParser/src/test/java/xplaneparser/entity/LineTypesTest.java
package xplaneparser.entity;

import static org.junit.Assert.assertFalse;
import static org.junit.Assert.assertTrue;

import org.junit.Test;

import ms.xplaneparser.entity.LineTypes;

public class LineTypesTest {

	
	   @Test
	   public void isTaxiwayTest() {
		   String taxicenterline="1";
		   assertTrue(LineTypes.isTaxiwayCenterLine(taxicenterline));
		   String taxicenterline2="101";
		   assertTrue(LineTypes.isTaxiwayCenterLine(taxicenterline2));
	   }
	   
	   
	   @Test
	   public void isNotTaxiwayTest() {
		   String taxicenterline="20";
		   assertFalse(LineTypes.isTaxiwayCenterLine(taxicenterline));
		   String taxicenterline2="";
		   assertFalse(LineTypes.isTaxiwayCenterLine(taxicenterline2));
	   }
}

File: XPlaneParser/src/test/java/xplaneparser/entity/LineTypesTest.kt
package xplaneparser.entity

import ms.xplaneparser.entity.LineTypes
import org.junit.Assert
import org.junit.Test

class LineTypesTest {
    @get:Test
    val isTaxiwayTest: Unit
        get() {
            val taxicenterline = "1"
            Assert.assertTrue(LineTypes.isTaxiwayCenterLine(taxicenterline))
            val taxicenterline2 = "101"
            Assert.assertTrue(LineTypes.isTaxiwayCenterLine(taxicenterline2))
        }

    @get:Test
    val isNotTaxiwayTest: Unit
        get() {
            val taxicenterline = "20"
            Assert.assertFalse(LineTypes.isTaxiwayCenterLine(taxicenterline))
            val taxicenterline2 = ""
            Assert.assertFalse(LineTypes.isTaxiwayCenterLine(taxicenterline2))
        }
}